flexget/plugins/sites/piratebay.py

```diff
@@ -151,7 +151,7 @@ def search(self, task, entry, config=None):
                     if size:
                         entry['content_size'] = parse_filesize(size.group(1))
                     else:
-                        log.error('Malformed search result? Title: "%s", No size? %s' % (entry['title'], sizeText))
+                        log.error('Malformed search result? Title: "%s", No size? %s', (entry['title'], sizeText))
```

Comment:
That's not gonna work. You need to remove the parenthesis around the variables. This is gonna crash since you're passing a tuple.